iohub/cli/cli.py

```diff
@@ -82,13 +82,22 @@ def info(files, verbose):
     is_flag=True,
     help="Arrange FOVs in a row/column grid layout for tiled acquisition",
 )
-def convert(input, output, format, scale_voxels, grid_layout):
+@click.option(
+    "--chunks",
+    "-c",
+    required=False,
+    default="XY",
+    help="Zarr chunk size given as 'XY', 'XYZ', or a tuple of chunk "
+    "dimensions.",
```

Comment:
Am I reading correctly that 'XYZ' chunks will be clipped to 500 MB? If yes, I think this is a good choice and should be documented:

```suggestion
    "dimensions. If 'XYZ', chunk size will be limited to 500 MB.",
```